lib/always_be_contributing/cli.rb

```diff
@@ -58,7 +58,8 @@ def render
            "github-org: #{github_org}",
            "date range: #{date_range}",
            '---------------------'
-      member_contribution_counts.each {|u| puts "%15s %3i" % u }
+      #member_contribution_counts.each {|u| puts ("%15s %3i\n" % u) rescue nil }
```

Comment:
Hey there! I found this branch randomly while cleaning up my machine. I'm not sure `#inspect` is actually better, needs verification.